merlin/models/tf/outputs/contrastive.py

```diff
@@ -314,6 +317,8 @@ def sample_negatives(
         return negatives
 
     def embedding_lookup(self, ids: tf.Tensor):
+        if isinstance(ids, tf.RaggedTensor):
+            ids = ids.to_tensor()
```

Comment:
The `Embedding` layer from Keras support ragged-tensors, right? What's the reasoning behind always converting to dense?